### ISSUE TITLE
ZHA light groups with switches ignore switch state

### ISSUE THREAD

**Issue by TheJulianJES** [zigpy/zha#330]
If a ZHA light group is created with multiple light members and a switch device, the light group state doesn't account for the switch state. Even if mixed types are in a light group, the on/off state of switches should still be used in the group state calculation.

Related HA issue: https://github.com/home-assistant/core/issues/124691

**Comment by dmulcahey** [comment]
I think this warrants discussion. What about fans? What happens when we expose groups for covers? Etc?

**Comment by TheJulianJES** [comment]
Yeah, it's just to keep track of this. I think it's only really realistic to combine switches and lights in ZHA groups.
Everything else seems very unlikely to do.
